styles.css

```diff
@@ -88,84 +98,191 @@ h3 {
     width: 60px;
     height: 60px;
     bottom: 80px;
-    right: 0;
+    border-radius: 5px;
+    right: 18px;
     z-index: 10;
 }
 
+.side-tab img {
+    position: absolute;
+    left: 7px;
+    top: 6px;
+}
+
+.side-tab:hover {
+    background-color: #ABB2B9;
+}
+
 /*=====================SECTION - ABOUT=======================*/
 
 section {
     min-height: 390px;
+    background-color: #EAECEE;
     position: relative;
 }
-.cloud-video {
-    position: absolute;
-    left: 0;
-    top: 0;
-    width: 100vw;
-    background-size: cover;
-    z-index: -1;
-}
 
 section div {
     padding: 70px 0 0;
     font-size: 1.5rem;
+    line-height: 150%;
     overflow: auto;
+    margin: 0 auto;
+    width: 90%;
+}
+
+.icons {
+    margin: 0 auto;
+    padding: 20px;
+}
+
+.icons div {
+    display: inline-block;
+    width: 19%;
+    height: 100%;
+}
+
+section img {
+    height: 100px;
+    width: 100px;
+    margin: 0 auto;
 }
 
 /*=====================MAIN - PORTFOLIO========================*/
 
 main {
+    width: 100%;
     overflow: auto;
-  padding-top: 60px;
 }
 
-main div {
-    width: 1600px;
+.portfolio-box {
+    width: 1700px;
     overflow: auto;
```

Comment:
do you need overflow:auto in both main selector and .portfolio-box selector ? could these be combined?

Reply:
you're right it wasn't needed in the main tag so i've removed it.